airbyte-integrations/connectors/source-jira/Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.3.7
+LABEL io.airbyte.version=0.3.8
```

Comment:
The version in the` metadata.yaml` file needs to be updated as well.

Reply:
done https://github.com/airbytehq/airbyte/pull/25798/files#diff-60f74e883cc95c68ef66f89e9e8e72f610300f153fde36acc1d70a1ec73d41ffR8